RecoParticleFlow/PFClusterProducer/python/particleFlowClusterOOTECAL_cff.py

```diff
@@ -6,7 +6,6 @@
 
 from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
 run2_miniAOD_80XLegacy.toModify(
-    particleFlowClusterOOTECAL.energyCorrector, 
-    recHitsEBLabel = "reducedEcalRecHitsEB",
-    recHitsEELabel = "reducedEcalRecHitsEE"
+    particleFlowClusterOOTECAL,
+    energyCorrector = dict(applyMVACorrections = True, maxPtForMVAEvaluation = 90., srfAwareCorrection = False, recHitsEBLabel = "reducedEcalRecHitsEB", recHitsEELabel = "reducedEcalRecHitsEE")
```

Comment:
please add linebreak for readability to keep the length within/below 100 chars.

Also, It looks like ```applyMVACorrections = True``` is redundant

Reply:
Hi Slava, yes that is right - removed applyMVACorrections = True.